test/disability-benefits/e2e/00.claims-list.e2e.spec.js

```diff
@@ -1,50 +1,46 @@
-if (!process.env.BUILDTYPE || process.env.BUILDTYPE === 'development') {
```

Comment:
I'm guessing we are ditching these now that we check buildtype at the `/script/run-nightwatch.js` level.  👍